packages/utils/src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { AccountUtils } from './account'
 export { AttestationUtils } from './attestations'
+export { DOLLAR_AMOUNT_FOR_ESTIMATE, CELO_AMOUNT_FOR_ESTIMATE } from './celoHistory'
```

Comment:
Sorry I wasn't clear, I meant putting these behind a wrapper object like for `AttestationUtils` above it. This update is equivalent to what was there before.

Reply:
I've decided to remove this export line because there are no methods we expose and we are importing the constants directly from `src/celoHistory`.